### PR TITLE
Re-worked phone generator method.

### PR DESCRIPTION
**Pull request by ocampd1**

### CODE REVIEW

File: chucks-pick3-web/angular-app/src/app/services/generate-customer.service/generate-customer.service.ts [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  }
+  public generatePhone() {
+    const areaCode =
+      [
+        214,
+        469,
+        972,
+      ];
+    const selectedAreaCode = areaCode[Math.floor(Math.random() * areaCode.length)];
+    const selectedPhoneNumberA = Math.floor(Math.random() * 999);
```
Comment (pouncilt) [inline chucks-pick3-web/angular-app/src/app/services/generate-customer.service/generate-customer.service.ts]: Nice Job!

File: chucks-pick3-web/angular-app/src/app/services/generate-customer.service/generate-customer.service.ts [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    const selectedPhoneNumberB = Math.floor(Math.random() * 9999);
```
Comment (pouncilt) [inline chucks-pick3-web/angular-app/src/app/services/generate-customer.service/generate-customer.service.ts]: Nice Job!

File: chucks-pick3-web/angular-app/src/app/services/generate-customer.service/generate-customer.service.ts (hunk 1 above)
Comment (ocampd1) [reply chucks-pick3-web/angular-app/src/app/services/generate-customer.service/generate-customer.service.ts]: Thank you, I appreciate it!

File: chucks-pick3-web/angular-app/src/app/services/generate-customer.service/generate-customer.service.ts (hunk 2 above)
Comment (ocampd1) [reply chucks-pick3-web/angular-app/src/app/services/generate-customer.service/generate-customer.service.ts]: Thank you!

**Pull request merged**